python/tritonserver/_c/tritonserver_pybind.cc

```diff
@@ -868,16 +844,15 @@ class PyInferenceResponse
     ThrowIfError(TRITONSERVER_InferenceResponseOutput(
         triton_object_, index, &name, &datatype, &shape, &dim_count, &base,
         &byte_size, &memory_type, &memory_type_id, &userp));
+    // The base pointer is deallocated when the response is deallocated.
```

Comment:
comment needed? are all pointers deallocated when response is deallocated?

Reply:
yes, the pointers are deallocated when the response is deallocated. cleaned up the comment: [Clean up base pointer deallocation comment](https://github.com/triton-inference-server/core/pull/426/commits/9575a0054b4510f161710aa60355a5d29c20c509)